wp-dependency-installer.php

```diff
@@ -253,7 +255,8 @@ public function admin_init() {
 
 			// Generate admin notices.
 			foreach ( $this->config as $slug => $dependency ) {
-				$is_required = $this->is_required( $dependency );
+				$is_required  = $this->is_required( $dependency );
+				$is_suggested = ! empty( $dependency['activate_notice'] );
```

Comment:
Not very sure if this could be the final version, but in this way it is possible to choose whether to display an "admin_notice" ( i.e. only for not yet installed plugins )

```js
"optional": true,
"activate_notice": false
```

Reply:
But why? There's already a notice about the plugin needing to be installed and potentially by which plugin. Any further information about why the plugin is desired rightfully belongs in the plugin readme.

Reply:
I close this due to outdated code